customer_outstanding_statement/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Customer Outstanding Statement',
+    'version': '9.0.1.0.0',
+    'category': 'Reports/pdf',
+    'summary': 'OCA Financial Reports',
+    'author': "Eficent, Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/account-financial-reporting',
```

Comment:
We have seen that the usual practice is to put the URL where the module can be found.
cc: @pedrobaeza